ONSdigital/dp-api-tests
# Feature/dataset observation endpoint

### What

Update dataset API  tests to include test for `.../observations` endpoint. Also updated request headers to simulate the requests in live (more realistic) and updated version/instance documents on setup to include the headers field and to have valid values.

### How to review

Make sure extension of acceptance tests make sense and the tests pass by doing the following:

* Run `dp-auth-api-stub`
* Make sure mongo db instance is running
* Make sure neo4j is running
* Run `dp-dataset-api` on branch `feature/observations-endpoint`
* Make sure `dp-api-tests` datasetAPI acceptance tests all pass, run `cd datasetAPI; go tests ./...` in root of repository

### Who can review

Team B
